### PR TITLE
Triage guide

### PR DESCRIPTION
**Pull request by TomAugspurger**
Discussed on the dev call yesterday.

cc @pandas-dev/pandas-core. Any sections I'm missing? Any statements you disagree with?

xref https://github.com/pandas-dev/pandas-governance/issues/11 (writing down a policy for triaging once we have one).

### CODE REVIEW

File: doc/source/development/maintaining.rst
Comment (datapythonista) [inline doc/source/development/maintaining.rst]: question

**Review by datapythonista — COMMENTED** [review]
great job, lgtm, just a typo

**Comment by alimcmaster1** [comment]
Awesome - great to have this info documented! @TomAugspurger

File: doc/source/development/maintaining.rst
Comment (jreback) [inline doc/source/development/maintaining.rst]: can you add the link to the https://stackoverflow.com/questions/tagged/pandas

File: doc/source/development/maintaining.rst
Comment (jreback) [inline doc/source/development/maintaining.rst]: @alimcmaster1 can you assign milestones as a triager?

**Review by jreback — APPROVED** [review]
really nice @TomAugspurger just some minor comments.

**Comment by TomAugspurger** [comment]
@alimcmaster1 a couple questions, since you're our only triager so far:

1. Can you close PRs, in additional to issues?
2. Can you push to PRs that aren't your own? I see a "Add more commits by pushing to ..." just above the CI checks.

File: doc/source/development/maintaining.rst
Comment (mroeschke) [inline doc/source/development/maintaining.rst]: ` label the issue as "Good First Issue" and "Needs Test" `

**Comment by alimcmaster1** [comment]
> @alimcmaster1 a couple questions, since you're our only triager so far:
> 
> 1. Can you close PRs, in additional to issues?
> 2. Can you push to PRs that aren't your own? I see a "Add more commits by pushing to ..." just above the CI checks.

1) Yes triage access allows you to close PRs
2) No I cannot push to PRs that aren't my own.

If helpful the full permissions matrix is here:
https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization

Thanks!

File: doc/source/development/maintaining.rst
Comment (alimcmaster1) [reply doc/source/development/maintaining.rst]: Yes you can - although I personally haven't done this much so far

File: doc/source/development/maintaining.rst
Comment (alimcmaster1) [inline doc/source/development/maintaining.rst]: Maybe handy to link: https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization

File: doc/source/development/maintaining.rst
Comment (alimcmaster1) [inline doc/source/development/maintaining.rst]: Different issue but we should add @simonjayhawkins and @topper-123 to this list if we still activity use it

Comment (TomAugspurger) [reply doc/source/development/maintaining.rst]: Done in https://github.com/pandas-dev/pandas-governance/pull/12

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: I would maybe add a "1. Thank the reporter for opening an issue"

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: Ideally, we should document the labels we use at the moment

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]:
```suggestion
   If the issue is clearly defined and the fix seems relatively straightforward,
```

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: Not sure if it fits here, or rather as an extra bullet point: has the issue a clear and descriptive title?

I think in the contributing guide there is somewhere a list of prefixes as well.

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]:
```suggestion
respond or self-close their issue if it's determined that the behavior is not a bug,
```

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: Something that I regularly do (and find valuable, also for afterwards writing simple tests), is further simplifying the example. Even if reporters provide a reproducible case, it can often be simplified a lot.

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: At some point, I think it would also be useful to have a more in-depth guide for reviewing pull requests (but not for this PR :))

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: Nitpick: a week can pass fast (certainly if people do this in their free time), maybe recommend two weeks?

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: If we are going to list what it means to be a maintainer / core team member, there is IMO missing one: think about API design or organisational issues, provide input on discussions, .. 

That might overlap with the line above, but I find it a bit narrow (it's not only relevant for stalled issues, although in those case the "final call" should probably come from maintainers)

File: doc/source/development/maintaining.rst
Comment (jorisvandenbossche) [inline doc/source/development/maintaining.rst]: In general (certainly if it's like a merge conflict or a small remaining lint error or typo fix), we should do this more often IMO instead of closing PRs that are in such a "final" state.

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks a lot for writing this down! Very nice guide and fluently written

**Comment by TomAugspurger** [comment]
Thanks. Hopefully addressed everything in 5772227e187ff0403cb432f85884b955a3495ae9

**Review by WillAyd — APPROVED** [review]
lgtm thanks @TomAugspurger

**Comment by jreback** [comment]
thanks @TomAugspurger very nice!

**Comment by alimcmaster1** [comment]
One thing I have noticed:

The triage team gives me access to cancel/restart build jobs on Travis but not on Azure. 

I can add this info if we think its worth it. (But I know we are hoping to migrate off travis soon)

Anyway just thought I would highlight

**Pull request merged**
